common/app/experiments/Experiments.scala

```diff
@@ -61,3 +62,10 @@ object Garnett extends Experiment(
   participationGroup= Perc0C
 )
 
+object ShowMoreExperiment extends Experiment(
```

Comment:
Should we name it `HideShowMoreButtonExperiment` to describe more explicitely what it does?

Reply:
I'll change it